### PR TITLE
Added post request service call, changing type logo, refactoring [dynamicTypeLogo]

### PR DESCRIPTION
**Pull request by jrgoodpasture**
### What was the feature/problem?

{Please write here}

### How does this PR address the above feature/problem?

{Please write here}

### Check lists (check `x` in `[ ]` of list items)

- [ ] Test passed
- [ ] Coding style (indentation, etc)

### Additional Comments (if any)

{Please write here}

### CODE REVIEW

File: ui/src/wishDetails/WishDetails.js [hunk 1]
```diff
@@ -1,30 +1,102 @@
...
   constructor(props) {
     super();
     this.state = {
-      id: 0
+      id: '',
+      wishDetails: {
+        id: '',
+        child: {
+          firstName: '',
+          lastName: ' ',
```
Comment (connorjones13) [inline ui/src/wishDetails/WishDetails.js]: is the space here desired?

File: ui/src/wishDetails/WishDetails.js [hunk 2]
```diff
@@ -1,30 +1,102 @@
...
+        image = Telescope;
+        break;
+      default:
+        break;
+    }
+    return image;
+  }
 
   render() {
+    const { child, details, sponsor } = { ...this.state.wishDetails }
```
Comment (connorjones13) [inline ui/src/wishDetails/WishDetails.js]: Not sure the additional object destructuring on the right hand side is necessary.

This should be sufficient
```javascript
const { child, details, sponsor } = this.state.wishDetails
```

File: ui/src/services/WishDetailsService.js [hunk 3]
```diff
@@ -6,10 +6,19 @@ const getWishDetails = async(id) => {
   return axios
     .get(`${expressDomain}/wishes/${id}`)
     .then(response => {
-      return response
+      return response.data
+    })
+}
+
+const makeAWish = async(wish) => {
+  return axios
```
Comment (connorjones13) [inline ui/src/services/WishDetailsService.js]: semi nit picky, but why mix the `.then()` syntax with async/await?

File: ui/src/wishDetails/WishDetails.js (hunk 2 above)
Comment (jrgoodpasture) [reply ui/src/wishDetails/WishDetails.js]: Yeah you’re right, I changed it while I was debugging to see something and forgot to change it back

File: ui/src/services/WishDetailsService.js (hunk 3 above)
Comment (jrgoodpasture) [reply ui/src/services/WishDetailsService.js]: Good point, was just due to the example I was taking from, I’ll fix in the morning

Comment (jrgoodpasture) [reply ui/src/services/WishDetailsService.js]: Fixed this in latest commit

File: ui/src/wishDetails/WishDetails.js (hunk 2 above)
Comment (jrgoodpasture) [reply ui/src/wishDetails/WishDetails.js]: Fixed this in latest commit

File: ui/src/wishDetails/WishDetails.js (hunk 1 above)
Comment (jrgoodpasture) [reply ui/src/wishDetails/WishDetails.js]: Fixed this in latest commit

Comment (jrgoodpasture) [reply ui/src/wishDetails/WishDetails.js]: Also changed to name instead of firstName, lastName

**Pull request merged**